src/seedu/addressbook/data/person/Address.java

```diff
@@ -11,8 +11,14 @@
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String SEPARATOR_COMMA_SPACE = ", ";
 
     public final String value;
+    public Block blk;
```

Comment:
Rename variables blk and pc.
Use full names. Refrain from using abbreviations when possible.